README.md

```diff
@@ -2,7 +2,7 @@
 
 ### What is it?
```

Comment:
If you want developers to use this, I would strongly encourage you add to the top:
* Some kind of logo, people love logos
* If there is any form of testing/code here, people love seeing integrated test badges. If this is docs, that might not be relevant for this case